packages/bcer-data-portal/app/src/views/Locations.tsx

```diff
@@ -582,6 +632,42 @@ export default function Locations() {
                       />                     
                     </Grid> 
                     </>}*/}
+                    <Grid item md={2} xs={6}>
+                      From
+                      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <KeyboardDatePicker
+                          className={classes.root}
+                          inputProps={{ className: classes.picker }}
+                          TextFieldComponent={TextFieldComponent}
+                          format="MM/dd/yyyy"
+                          value={selectedFromDate ? moment(selectedFromDate) : null}
+                          onChange={onFromDateChange}
+                          showTodayButton={true}
+                          clearable={true}
+                          KeyboardButtonProps={{
+                            'aria-label': 'change date',
+                          }}
+                        />
+                      </MuiPickersUtilsProvider>
+                    </Grid>   
+                    <Grid item md={2} xs={6}>
+                      To
+                      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <KeyboardDatePicker
+                          className={classes.root}
+                          inputProps={{ className: classes.picker }}
+                          TextFieldComponent={TextFieldComponent}
+                          format="MM/dd/yyyy"
+                          value={selectedToDate ? moment(selectedToDate) : null}

```

Comment:
You are using null here, so the previous review should probably be null not "" but please confirm from documentation and adjust accordingly